subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/internal/dependencies/PrintAppsDependenciesGraphTask.kt

```diff
@@ -65,11 +65,14 @@ internal abstract class PrintDependenciesGraphTask @Inject constructor(
             }
     }
 
+    private fun graphBuilder(): DependenciesGraphBuilder =
+        DependenciesGraphBuilder(project, GradleLoggerFactory.fromTask(this))
```

Comment:
I don't know If it's important, but we should reuse `DependenciesGraphBuilder` because of cache inside it

Reply:
Sure, in general, we need to do it before reusing builder in other modules.
In this task this is ok, it's mainly for testing.